apps/user-profile/src/app/components/header/header.component.html

```diff
@@ -7,6 +7,39 @@
   <a class="ml-3" [innerHTML]="logo" [routerLink]="disableLogo ? [] : ['/']" queryParamsHandling="merge"></a>
 
   <p [ngStyle]="{'color': textColor}" class="ml-3 mt-auto mb-auto">{{label}}</p>
-</mat-toolbar>
 
+  <div id="nav-menu-user-info" class="justify-content-end">
+    <button mat-button [matMenuTriggerFor]="menu">
+      <mat-icon [ngStyle]="{'color': iconColor}">
+        person
+      </mat-icon>
+      <span [ngStyle]="{'color': textColor}" class="d-none d-lg-inline">
+        {{principal.user.titleBefore}}
+        {{principal.user.firstName}}
+        {{principal.user.middleName}}
+        {{principal.user.lastName}}
+        {{principal.user.titleAfter}}
+      </span>
+    </button>
+    <mat-menu #menu="matMenu">
+      <button mat-menu-item *ngIf="logoutEnabled" (click)="onLogOut()">{{'HEADER.LOG_OUT' | translate}}</button>
+    </mat-menu>
 
+    <button
+      mat-icon-button
+      [matMenuTriggerFor]="appsMenu"
+      *ngIf="activeLink && adminGuiUrl"
+      [matTooltip]="'HEADER.OTHER_APPLICATIONS' | translate">
+      <mat-icon [ngStyle]="{'color': iconColor}" class="app-icon">
+        apps
+      </mat-icon>
+    </button>
+    <mat-menu #appsMenu="matMenu">
+      <button
+        mat-menu-item
+        (click)="redirectToAdminGUI()">
+        <span class="black">{{adminLabel}}</span>
+      </button>
+    </mat-menu>
```

Comment:
Would it be possible to move this component (or some part of it) to the shared libraries? Since we use it in two applications.

Reply:
This block of code has been moved to the separated shared component.